rudder-inventory-ldap/SOURCES/rudder-inventory-ldap.init

```diff
@@ -174,6 +174,7 @@ BACKUP_SUFFIX="`date +%Y%m%d%H%M%S`.ldif"
 # Other
 TIMEOUT="60"    # Max time to stop process
 FD_LIMIT="2048" # Max file descriptor
+VM_LIMIT="108490720" # Max virtual memory
```

Comment:
This is the init script, every thing is done here.
So you could just put ulimit at the beginning of the script and forget everything else. 
And it is probably necessary to put the same command in the postinstall script that calls slapcat et al.